### ISSUE TITLE
Suggestion: view HP bar on same screen where you use healign potions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#179]
```
As it stands, you can't see your HP while using potions, you need to flip btw 
screens. I spam minor healing potions, so seeing the bar would save me some 
unnecessary thinking (because who needs THAT in videogames? :P )

The quick-item button would do the trick.
```

Original issue reported on code.google.com by `jch...@email.wm.edu` on 8 Apr 2011 at 5:18

**Comment by GoogleCodeExporter** [comment]
```
Good point. Adding quickitem slots to the main screen would be preferable IMO, 
because then you wonldn't even need to go into inventory to access them.
```

Original comment by `oskar.wi...@gmail.com` on 5 May 2011 at 9:35

**Comment by GoogleCodeExporter** [comment]
```
In my opinion, this is now solved with the excellent quickitem slots that 
Olivier did.

By using items from the quickslots, you can still see the regular healthbar.

Should we close this one?
```

Original comment by `oskar.wi...@gmail.com` on 22 Jun 2011 at 8:26
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
yes.
```

Original comment by `SamuelPl...@gmail.com` on 22 Jun 2011 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
Marking this as Ready for v0.6.9 then.
```

Original comment by `oskar.wi...@gmail.com` on 23 Jun 2011 at 6:48
- Changed state: **Ready**
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:51
- Changed state: **Done**
